packages/ogg/ogg.0.7.4/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings to libogg"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exceptio"
```

Comment:
```suggestion
license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
```
I'll merge this before merging the PR, after the CI has finished

Reply:
Thanks, The typo is fixed upstream.